inmap_test.go

```diff
@@ -580,7 +580,14 @@ func TestConverge(t *testing.T) {
 			t.Logf("%s completed after %d iterations.", convergenceNames[i], iterations)
 		}
 
-		r, err := d.Results(false, true, map[string]string{"PrimPM25": "PrimaryPM25"})
+		o := NewOutputter("", false, map[string]string{"PrimPM25": "PrimaryPM25"}, nil)
+
+		err := o.CheckForDerivatives()
+		if err != nil {
+			t.Error(err)
+		}
+
+		r, err := d.Results(o.AllLayers, o.OutputVariables, o.GetVariables, o.OutputFunctions)
```

Comment:
I didn't realize that you could do that. But also, there is one additional parameter in the Outputter struct that is not an input to the Results function (o.fileName).